### PR TITLE
Update ru.txt

### PR DESCRIPTION
**Pull request by geoffthemedio**
Resolves conflicts of and supercedes #2793

### CODE REVIEW

**Comment by o01eg** [comment]
I remember I left some notes about previous PR. Also it's a still very big PR to be correctly reviewed.

**Comment by geoffthemedio** [comment]
> I remember I left some notes about previous PR. Also it's a still very big PR to be correctly reviewed.

It looked like there were replies to all your comments, with no additional follow up or re-reply for some time...

I'd like to merge this before the next release, but will certainly wait for your approval.

**Comment by Cjkjvfnby** [comment]
> Also it's a still very big PR to be correctly reviewed.

Technically it is easy to split PR to separate PRs because translation entries are not related(almost).  I can help @naiffuR  with it.     

@o01eg what do you think?

File: default/stringtables/ru.txt [hunk 1]
```diff
@@ -1274,6 +1414,42 @@ OPTIONS_DB_MP_HOST_ADDRESS
...
+Включить отправку статики империи игроку.
+
+OPTIONS_DB_FIRST_TURN_TIME
+Абсолютный момент времени в формате «2019-01-20 11:59:59» в UTC. Если пусто, продвижение первого хода произойдет после истечения интервала. Требуется фиксированный интервал для включения.
+
+OPTIONS_DB_TIMEOUT_INTERVAL
+Максимальный интервал в секундах между ходами вперед. Если 0, то ходы по тайм-ауту отключены, и ходы будут продвигаться только после того, как все игроки закончили свой ход..
+
+OPTIONS_DB_TIMEOUT_FIXED_INTERVAL
+Ход вперед через фиксированные промежутки времени, независимо от того, когда и когда игроки отправили приказы хода.
```
Comment (o01eg) [inline default/stringtables/ru.txt:1448]: @naiffuR `когда и когда`

**Review by o01eg — CHANGES_REQUESTED** [review]

**Comment by o01eg** [comment]
@Cjkjvfnby Yes, splitting PR would be nice.

File: default/stringtables/ru.txt (hunk 1 above)
Comment (naiffuR) [reply default/stringtables/ru.txt:1448]: fix: когда и какой игрок отправил приказ хода

**Comment by naiffuR** [comment]
Fix [ru.txt](https://github.com/freeorion/freeorion/files/4562892/ru.txt)

**Comment by geoffthemedio** [comment]
> Fix [ru.txt](https://github.com/freeorion/freeorion/files/4562892/ru.txt)

Not sure what you mean...

**Pull request closed without merging**